README.md

```diff
@@ -3,33 +3,70 @@
 [Github Action](https://github.com/features/actions) to set up an environment
 for building and testing PHP extensions on Windows.
 
-## Example Usage
+## Inputs
+- `version`: the PHP version to build for
+  (`7.0`, `7.1`, `7.2`, `7.3`, `7.4`, `8.0` or `8.1`)
```

Comment:
1. Pre-existing: this should include 8.1
2. This should be quoted, maybe, i.e. `'7.0'`, because (1) there might someday have php '8.10' or (2) there might